README.md

```diff
@@ -82,11 +83,11 @@ This will enable the HTTPS ingresses of the Kubernetes cluster to use the fore-m
 **Note:**
 From v1.1.0 onwards, certificates can have user-defined names. Each service could use its own certificate and there does not need to be one common certificate used across services.
 
-## Upgrade CPI
```

Comment:
Is upgrade to the latest CPI version not supported?

Reply:
@arunmk would you please comment if we will document the upgrade to CPI 1.2?